ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/cco-manual-users-static/provision/cucushift-installer-rehearse-gcp-ipi-cco-manual-users-static-provision-chain.yaml

```diff
@@ -1,14 +1,22 @@
 chain:
   as: cucushift-installer-rehearse-gcp-ipi-cco-manual-users-static-provision
   steps: 
-  - chain: ipi-conf-gcp
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-gcp
```

Comment:
Can we keep `chain: ipi-conf-gcp` here ?

Reply:
The chain has the step `ipi-install-monitoringpvc` which would lead to cluster operator "monitoring" degraded (see Slack [discussion](https://redhat-internal.slack.com/archives/CH76YSYSC/p1727085466509049?thread_ts=1727079476.858529&cid=CH76YSYSC)). That's it, thanks!